tendrl-monitoring-integration.spec

```diff
@@ -25,6 +26,8 @@ Python module for Tendrl to create a new dashboard in Grafana
 
 %prep
 %setup
+unzip %SOURCE1
```

Comment:
We need to build this plugin as a subpackage as in https://github.com/ceph/cephmetrics/blob/master/cephmetrics.spec.in#L33
package name should be tendrl-grafana-plugins